### PR TITLE
Refactoring Sealights and AppDynamics Hooks

### PR DESCRIPTION
**Pull request by liornabat-sealights**
Thanks for contributing to the buildpack. To speed up the process of reviewing your pull request please provide us with:

* A short explanation of the proposed change:
This PR refactor PR 809 per @robdimsdale request

* An explanation of the use cases your change solves

* [ x] I have viewed signed and have submitted the Contributor License Agreement

* [ x] I have made this pull request to the `develop` branch

* [x ] I have added an integration test

### CODE REVIEW

File: src/python/hooks/sealights.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ func (sh SealightsHook) BeforeCompile(stager *libbuildpack.Stager) error {
 		return nil
 	}
 
-	sealgithsServiceName, sealightsPlan := getSealightsServiceName(services, sh.Log)
+	sealgithsServiceName, sealightsPlan := getSealightsServiceName(services)
```
Comment (robdimsdale) [inline src/python/hooks/sealights.go:102]: I think we still have typos here: `sealgiths` -> `sealights`.

**Review by robdimsdale — COMMENTED** [review]
Looks great - one remaining typo and then I think we can merge.

**Comment by robdimsdale** [comment]
@liornabat-sealights thank you for submitting this so quickly! and for fixing the app dynamics integration test too. I really appreciate it.

**Comment by robdimsdale** [comment]
I fixed the last remaining typo. I'll enable auto-merge, and we can release once the PR is merged.

**Review by brayanhenao — APPROVED** [review]

**Pull request merged**